src/NBitcoin/Money.cs

```diff
@@ -11,10 +11,10 @@ public static class MoneyExtensions
     {
         public static Money Sum(this IEnumerable<Money> moneys)
         {
-            if(moneys == null)
+            if (moneys == null)
```

Comment:
I can't find what has changed in this class `Money`

Reply:
@dangershony there was an empty constructor added in an earlier commit, this has been reverted.